jcl/src/java.management/share/classes/java/lang/management/MemoryNotificationInfo.java

```diff
@@ -89,9 +89,17 @@
 	 *            {@link #MEMORY_COLLECTION_THRESHOLD_EXCEEDED} notifications,
 	 *            this will be the number of times that the collection usage
 	 *            threshold was passed.
+	 * @throws NullPointerException
+	 *            when poolName or usage is null
 	 */
 	public MemoryNotificationInfo(String poolName, MemoryUsage usage, long count) {
-		super();
+		if (poolName == null)
```

Comment:
Our standard is to use { } brackets, to avoid problems in the future if the code is modified.